portfolio/src/main/webapp/AboutMe.html

```diff
@@ -6,9 +6,9 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:#FEFBE9;">
     <div id="content">
-      <a href = "/index.html"><img src="/images/Home_Button.png"/></a>
+      <a href = "/index.html"><img src="/images/Home_Button.png"width="50" height="50"/></a>
```

Comment:
We typically use CSS to scale images rather than using the width and height attributes.

You could add a class or ID to these, include a selector for that in your styles.css and apply `width: 50px` and `height: 50px` there.